### PR TITLE
Replace msg.sender with _msgSender()

### LINKED ISSUE #19: Replace msg.sender with _msgSender

### PR DESCRIPTION
**Pull request by vrde**
Close #19

### CODE REVIEW

File: contracts/ResolutionManager/ResolutionManager.sol [hunk 1]
```diff
@@ -130,7 +131,7 @@ contract ResolutionManager is AccessControl {
         require(
             _shareholderRegistry.isAtLeast(
                 _shareholderRegistry.CONTRIBUTOR_STATUS(),
-                msg.sender
+                _msgSender()
```
Comment (sirnicolaz) [inline contracts/ResolutionManager/ResolutionManager.sol:134]: Do we need to add a further dependency to the contracts to add a functionality that we will never use?

**Review by sirnicolaz — CHANGES_REQUESTED** [review]
See comment

File: contracts/ResolutionManager/ResolutionManager.sol (hunk 1 above)
Comment (vrde) [reply contracts/ResolutionManager/ResolutionManager.sol:134]: Exactly

**Review by sirnicolaz — APPROVED** [review]
The motivations of the committer were sufficiently convincing.

**Pull request merged**